convert2rhel/unit_tests/actions/system_checks/efi_test.py

```diff
@@ -161,21 +174,31 @@ def test_check_efi_efi_detected_nofile_entry(self):
                         "The current UEFI bootloader '0002' is not referring to any binary UEFI"
                         " file located on local EFI System Partition (ESP)."
                     ),
-                    remediation=None,
+                    remediation="",
```

Comment:
Did this pass before? 🤔

Reply:
Yeah, it was.  I looked into it and the reason is that we're using a real `actions.ActionMessage` here.  That class makes sure to turn `None` values for parameters into empty string since it is later used in the json data which we will pass on to the Tasks App.